openapi/bitly/Package.md

```diff
@@ -0,0 +1,19 @@
+Connects to [Bitly API v4.0.0](https://dev.bitly.com/api-reference) from Ballerina
+
+## Package overview
+The `ballerinax/bitly` is a [Ballerina](https://ballerina.io/) connector for Bitly API.
+This package provides the capability to access c API.
```

Comment:
Please add the name of the API here.

Reply:
Hi, Did the suggested changes. 
/Thanks